Makefile

```diff
@@ -4,10 +4,10 @@ build:
 	docker build -t daymet .
 
 test:
-	docker run --rm -v "${PWD}":/tmp daymet loyalty_degauss.csv
+	docker run --rm -v $PWD:/tmp ghcr.io/degauss-org/daymet:0.1.4 loyalty_degauss.csv --vars=tmax

```

Comment:
This step is intended to test the container as it is built. Changing to `ghcr.io/degauss-org/daymet:0.1.4` will test the already published version stored in github container registry. Using the short name (`daymet`) tests the most recently built container, which is what we want here.